bundle/manifests/projectcontour-contour-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,350 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+    operators.operatorframework.io/builder: operator-sdk-v1.8.0+git
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
+  name: projectcontour-contour-operator.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: BackendPolicy
+      name: backendpolicies.networking.x-k8s.io
+      version: v1alpha1
+    - kind: Contour
+      name: contours.operator.projectcontour.io
+      version: v1alpha1
+    - kind: ExtensionService
+      name: extensionservices.projectcontour.io
+      version: v1alpha1
+    - kind: GatewayClass
+      name: gatewayclasses.networking.x-k8s.io
+      version: v1alpha1
+    - kind: Gateway
+      name: gateways.networking.x-k8s.io
+      version: v1alpha1
+    - kind: HTTPProxy
+      name: httpproxies.projectcontour.io
+      version: v1
+    - kind: HTTPRoute
+      name: httproutes.networking.x-k8s.io
+      version: v1alpha1
+    - kind: TCPRoute
+      name: tcproutes.networking.x-k8s.io
+      version: v1alpha1
+    - kind: TLSCertificateDelegation
+      name: tlscertificatedelegations.projectcontour.io
+      version: v1
+    - kind: TLSRoute
+      name: tlsroutes.networking.x-k8s.io
+      version: v1alpha1
+    - kind: UDPRoute
+      name: udproutes.networking.x-k8s.io
+      version: v1alpha1
+  description: Projectcontour Contour Operator description. TODO.
```

Comment:
I don't know whether the description is not set because it's WIP or not :)

Reply:
As per my previous comment